client/blocks/login/index.jsx

```diff
@@ -850,7 +863,7 @@ export default connect(
 			dispatchProps.sendEmailLogin( stateProps.usernameOrEmail, {
 				redirectTo: stateProps.redirectTo,
 				loginFormFlow: true,
-				showGlobalNotices: true,
+				showGlobalNotices: false,
```

Comment:
This is to prevent toast notifications on the top right. I think it looks less distracting to the user since the button now has animation.